src/sources/aws_s3/sqs.rs

```diff
@@ -459,6 +462,7 @@ impl IngestorProcess {
         let object = object_result?;
 
         let metadata = object.metadata;
+        // TODO: Insert last_modified as source metadata?
```

Comment:
Multiple TODOs added in the diff- are they flags added to resolve during PR review discussion or something that we will need to follow-up on in future changes ?

Reply:
👀 from the summary:
> It does add two TODOs that may insert an incorrect ingest_timestamp. These TODOs can be resolved as part of this PR or separately.

Reply:
:smile: Right so I guess to rephrase- I don't have the answers to resolve that as part of the PR but, if we are doing that separately- do any of the (four?) TODOs require a GH issue to reference in the code comment?

Reply:
Ah, so - yes this should _at least_ be done as part of the `log_namespace` epic (meaining new issues), or done here before I merge.

I think the question I actually have is if we change the mapping from `Legacy` => `Vector` as
`.timestamp` => `%aws_s3.timestamp` (source metadata) and have a "new" `%vector.ingest_timestamp` that would always be `now()`. If we do this we should also probably decide if _both_ of these would be `now()` we'd only include the `%vector` timestamp.